src/main/java/org/jitsi/videobridge/LipSyncHack.java

```diff
@@ -434,6 +445,107 @@ public void onRTPTranslatorWillWriteVideo(
     }
 
     /**
+     * {@inheritDoc}
+     */
+    @Override
+    public PacketTransformer getRTPTransformer()
+    {
+        return this;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public PacketTransformer getRTCPTransformer()
+    {
+        return null;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void close()
+    {
+
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public RawPacket[] reverseTransform(RawPacket[] pkts)
+    {
+        return pkts;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public RawPacket[] transform(RawPacket[] pkts)
+    {
+        if (pkts == null || pkts.length == 0)
+        {
+            return pkts;
+        }
+
+        RawPacket[] extra = null;
+
+        for (int i = 0; i < pkts.length; i++)
+        {
+            if (pkts[i] == null)
+            {
+                continue;
+            }
+
+            long ssrc = pkts[i].getSSRCAsLong();
+            if (!ssrcsWithoutBlackKeyframes.isEmpty()
```

Comment:
so is the idea that this logic only handles sending the very first black keyframe and doesn't ever kick in again?  just trying to get a sense of how frequently we'll have to do that array copy below